### DISCUSSION TITLE
Embeddable Rowy Table

### DISCUSSION THREAD

**Discussion by aaazzam** [AntlerVC/firetable#963] [Idea / Feature Request]
**Is your feature request related to a problem? Please describe.**
The default install (which is really well documented!) launches a platform-view, whose UI is ideal for internal use. I would like to use firetable in a product, in which I assign every new user a default firetable embedded in an existing user portal. 

**Describe the solution you'd like**
 I would love to take an existing table and similarly be able to embed a stand-alone table. This functionality is present in something like firepad, which lets you embed a standalone rich text editor.

**Describe alternatives you've considered**
The editor version of datatables.net is my current working alternative. However, this doesn't allow for collaborative editing of tables like firetable.

**Comment by notsidney** [comment]
Thanks for the idea @AAAZZAM! This looks to be in line with the work needed to implement #228

**Comment by rubenheymans** [comment]
Too bad I thought this would be possible. I have my own backend application written in Angular and I want to show the datagrids on the pages in my app. So my client can login to one backend and manage everything from there. Is there an update for this already?
